lms/services/lti_grading/interface.py

```diff
@@ -0,0 +1,33 @@
+class LTIGradingService:  # pragma: no cover
```

Comment:
I ended up not using a dataclass for these

- Being a bit pedantic this is not "data". This is very debatable but it's just my preference here.
- More concretely we've seen before problems when the dataclass magic interfired with other decorators. For example for `lru_cache` you need to make the dataclass hasheable. Not the end of the world but having the manual `__init__` is not horrible either.